packages/lodestar-beacon-state-transition/test/perf/phase0/fast/slot/slots.test.ts

```diff
@@ -27,7 +27,7 @@ describe("Process Slots Performance Test", function () {
     const start = Date.now();
     phase0.fast.processSlots(epochCtx, state, state.slot + numSlot);
     logger.profile(`Process ${numSlot} slots`);
-    expect(Date.now() - start).lt(1100);
+    expect(Date.now() - start).lt(570);
```

Comment:
with new ssz + this PR, the new epoch transition takes 500ms - 600ms in average instead of ~1100ms as previously